src/fsharp/service/ServiceLexing.fsi

```diff
@@ -4,6 +4,8 @@ namespace FSharp.Compiler.SourceCodeServices
 
 open FSharp.Compiler 
 
+#nowarn "57"
```

Comment:
Could you add a comment about which warning was suppressed?

Reply:
Sure